examples/data-transfer-pg/sql/db2_setup.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE fruits (id INT, name VARCHAR(50), quantity INT);
```

Comment:
Comments are must to make code descriptive and informative.
```suggestion
--
-- Create table for fruits data (destination database).
CREATE TABLE fruits (id INT, name VARCHAR(50), quantity INT);
```